pkg/controllers/utils/pod.go

```diff
@@ -52,8 +52,9 @@ func PodIsScheduled(pod *v1.Pod) bool {
 }
 
 func PodIsTerminated(pod *v1.Pod) bool {
```

Comment:
What about changing func name to something like `PodHasNoRunningContainer`?

Reply:
done~